Specs/DateTime/German/MergedParser.json

```diff
@@ -76,29 +76,74 @@
     "Context": {
       "ReferenceDateTime": "2016-11-07T00:00:00"
     },
-    "Comment": "Returning only the future resolution",
-    "NotSupported": "dotnet, javascript, python",
+    "Comment": "Returning only the future resolution, which is sematically correct in that case",
+    "NotSupported": "javascript, python",
+    "Results": [
+      {
+        "Text": "Freitag Nachmittag",
+        "Type": "datetimeV2.datetimerange",
+        "Value": {
+          "values": [
+            {
+              "timex": "2016-11-11TAF",
+              "type": "datetimerange",
+              "start": "2016-11-11 12:00:00",
+              "end": "2016-11-11 16:00:00"
+            }
+          ]
+        },
+        "Start": 3,
+        "Length": 18
+      }
+    ]
+  },
+  {
+    "Input": "Letzten Freitag Nachmittag habe ich Feierabend gemacht",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "Comment": "Returning future resolution instead of past, seems to work for different reference date, see below.",
+    "NotSupported": "dotNet, javascript, python",
```

Comment:
What do you mean "seems to work for different reference date"? If this works correctly, why marked as NotSupported in dotnet?

Reply:
The test case with "ReferenceDateTime": "2016-11-07T00:00:00" returns a future date, the next with "ReferenceDateTime": "2018-12-09T00:00:00" returns a past reference date. In the context of the sentence the past date would be correct, but I don't understand the behavior. Shouldn't this always return behave the same or return future and past? "Letzten Freitag" is "Last Friday" in the sentence.